src/main/drivers/pwm_output.h

```diff
@@ -27,6 +27,8 @@ typedef enum {
     PWM_TYPE_ONESHOT42,
     PWM_TYPE_MULTISHOT,
     PWM_TYPE_BRUSHED,
+    PWM_TYPE_DSHOT1200,
```

Comment:
@martinbudden done :) Might as well get it how we want if it is dependent on a configurator change